samcli/lib/init/template_modifiers/xray_tracing_template_modifier.py

```diff
@@ -22,7 +23,19 @@ class XRayTracingTemplateModifier(TemplateModifier):
         "https://github.com/awslabs/serverless-application-model/blob/master/docs/globals.rst\n"
     )
 
-    def _add_new_field_to_template(self):
+    def _get_template(self) -> Any:
```

Comment:
should the type hint be `List[str]` instead?

Reply:
return type of yaml.load() is not List(str)

Reply:
This is under `XRayTracingTemplateModifier`, so it's not relevant to what the `_get_template` is returning under the `ApplicationInsightsTemplateModifier` class.

Reply:
Sure, will change this